prow/plugins/trickortreat/trickortreat.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package trickortreat adds halloween images to an issue or PR in response to a /trick-or-treat comment
+package trickortreat
```

Comment:
consider `trick-or-treat` for the package and plugin name

Three unseparated words is IMO too many, (see merge-mmethod-comment, owners-label, etc plugins, external-plugins, pod-utils packages)

Reply:
changed the plugin name. Go doesn't seem like dash in package name, so keep it as is